compiler/luci-interpreter/src/Interpreter.cpp

```diff
@@ -201,19 +217,14 @@ void Interpreter::interpret()
   {
     const auto *node = loco::must_cast<const luci::CircleNode *>(loco_node);
 
-    // Compute the result for the node. CircleConst, CircleInput and CircleOutput nodes
-    // are auxiliary, there is nothing to compute for them.
-    if (node->opcode() != luci::CircleOpcode::CONST &&
-        node->opcode() != luci::CircleOpcode::CIRCLEINPUT &&
-        node->opcode() != luci::CircleOpcode::CIRCLEOUTPUT)
+    if (isExecutableNode(node))
     {
       Kernel *kernel = _kernel_map->getKernel(node);
       kernel->execute();
     }
 
-    // Notify the observers that the node's output tensor has changed. This is not done
-    // for CircleOutput nodes because they do not produce any tensors.
-    if (node->opcode() != luci::CircleOpcode::CIRCLEOUTPUT)
+    // Notify the observers that the node's output tensor has changed.
+    if (isTensorProducingNode(node))
```

Comment:
Maybe call observers after all nodes, including split itself?
For example this could be useful for profiling.

Reply:
Ok, I got it. Method name is `postTensorWrite`, now it make sence